style.css

```diff
@@ -15,3 +15,19 @@ body {
 .nav-bar {
   padding-top: 45px;
 }
+
+.faq {
+  font-family: 'roboto', sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  color: #728993;
+  margin-right: 40px;
+}
+
+.security {
+  font-family: 'roboto', sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  color: #728993;
+  margin-right: 21px;
+}
```

Comment:
nice job here, but you have the exact same styles for both `.faq` and `.security`. You can just add that rule block to a `. navbarlinktext` selector instead.